packages/docs/src/components/header/components/HeaderSelector.vue

```diff
@@ -35,11 +41,14 @@ export default {
     },
   },
 
-  setup () {
+  setup (props: any, { emit }: any) {
```

Comment:
Wrap the exported config with a `defineComponent`, and we will no need to use any type here.